client/src/GettingStarted/GettingStarted.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { Button, Card, Col, Row, Typography } from 'antd'
+import Icon from '@material-ui/core/Icon'
+import AssignmentIcon from '@material-ui/icons/Assignment'
+import BusinessIcon from '@material-ui/icons/Business'
+import CloudUploadIcon from '@material-ui/icons/CloudUpload'
+import PlaylistAddIcon from '@material-ui/icons/PlaylistAdd'
+
+export function GettingStarted() {
+  /*
+  TODO:
+  - layout/general
+    - ask about outer layout
+    - change classnames to suit P4P
+    - double-check responsive breakpoints
+  - content
+    - replace "Amanda" with user name
```

Comment:
+1 and this leads to a maybe longer conversation about user data; right now we're storing user tokens in local storage and nothing else.  Do we want to store the user's `greeting_name` as well so that this page specifically doesn't have to make an api call?  I'm not super opinionated about frontend so I'd love some opinions on this.  cc @sassygrody @noragharris